pkg/policy/api/rule.go

```diff
@@ -62,6 +62,30 @@ type Rule struct {
 	Description string `json:"description,omitempty"`
 }
 
+// Entity specifies a special entity for rules
```

Comment:
Entity specifies the class of receiver/sender endpoints that do not have individual identities. Entities are used to describe "outside of cluster", "host", etc.

Reply:
will do